### PR TITLE
passing add-in url via dist gulp task using --url parameter

### PR DESCRIPTION
**Pull request by ministainer**
passing add-in url via dist gulp task using --url parameter 
examples :
`gulp dist --url https://myserver.local/`
`gulp dist --url https://myaddin.azurewebsites.net/`

### CODE REVIEW

**Comment by ministainer** [comment]
Hi @jthake ,
I know you might not have the time , but any feedback will be much appreciated :+1: 

Amr

**Comment by ministainer** [comment]
@beth-panx  I see that this repo got more love recently which is great , I have this PR sitting still for around 4 months now .

Amr

**Comment by beth-panx** [comment]
Hi @ministainer I am currently working on reviewing pull requests as well as closing issues.. We are currently on the stage of prioritizing some of the works that needs to be done. This is def on my task list :)

File: generators/content/templates/common/gulpfile.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 'use script';
 
 var gulp = require('gulp');
+var replace=require('gulp-replace');
+var argv = require('yargs').argv;
```
Comment (beth-panx) [inline generators/content/templates/common/gulpfile.js]: Should this package be added to package.json?

Comment (ministainer) [inline generators/content/templates/common/gulpfile.js]: @beth-panx it should let me add it and update tests as well

**Pull request merged**